c/src/detectors/c-redundant-free-usage/c-redundant-free-usage_non-compliant.c

```diff
@@ -0,0 +1,14 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// {fact rule=c-redundant-free-usage@v1.0 defects=1}
+#include <stdlib.h>
+
+int noncompliant() {
+    //NonCompliant : Calling `free(var)` twice on the same memory address, which may leads to program crashes.
```

Comment:
1. Should be `... which may lead to program crashes.`
2. The comment seems to be misplaced